samples/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required( VERSION 2.6.2 FATAL_ERROR )
```

Comment:
I don't think that's a good idea to go below cmake required by mediasdk itself. And we check 2.8.5 on our side, right?